### PR TITLE
New package: arp-scan-1.9

### PR DESCRIPTION
**Pull request by faulesocke**
A small utility to discover and fingerprint hosts on the local network.

### CODE REVIEW

File: srcpkgs/arp-scan/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'arp-scan'
+pkgname=arp-scan
+version=1.9
+revision=1
+build_style=gnu-configure
+makedepends="automake libpcap-devel"
```
Comment (Johnnynator) [inline srcpkgs/arp-scan/template]: automake should be in hostmakedepnds since it is needed for the build host.

**Comment by faulesocke** [comment]
Okay, I've changed the commit, automake is now in hostmakedepends.

File: srcpkgs/arp-scan/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+license="GPL-3"
+homepage="http://www.nta-monitor.com/tools/arp-scan/"
+distfiles="https://github.com/royhills/$pkgname/archive/$version/$pkgname-$version.tar.gz"
+checksum=b2a446a170e4a2feeb825cd08db48a147f8dffae702077f33e456c4200e7afb0
+
+pre_configure() {
+	autoreconf --install
+}
+
+# vim: set sw=4 ts=4 noet:
```
Comment (Vaelatern) [inline srcpkgs/arp-scan/template]: no.

Comment (faulesocke) [reply srcpkgs/arp-scan/template]: Sorry, I don't understand?

Comment (Johnnynator) [reply srcpkgs/arp-scan/template]: He is probly speaking about you vim line.

Comment (faulesocke) [reply srcpkgs/arp-scan/template]: Hmm, I grep'ed for vim modelines before commiting and found a few other templates containing some so I thought it's not a problem. If it is, I can of course remove it.

**Comment by faulesocke** [comment]
Any progress? What should be changed in the template to get it merged?

**Comment by Johnnynator** [comment]
You could either mark armv7* as broken or fix it, otherwise it looks fine to me.

**Comment by faulesocke** [comment]
Fixed cross compilation by backporting a newer commit.

**Pull request closed without merging**